util/config.go

```diff
@@ -329,17 +334,44 @@ func (config *Config) getAndroidManifestPath() string {
 }
 
 func (config *Config) getAndroidConfigModelPath() string {
-	return path.Join(getPlatformSpecificAndroidClasspathPath(config), "OneginiConfigModel.java")
+	modelPath := path.Join(getPlatformSpecificAndroidClasspathPath(config), "OneginiConfigModel.java")
+	// if modelPath has no package name, check namespace property in build.gradle
+	if strings.HasSuffix(modelPath, "java/OneginiConfigModel.java") {
+		modelPath = strings.TrimSuffix(modelPath, "OneginiConfigModel.java")
+		modelPath = path.Join(modelPath, strings.ReplaceAll(config.getAndroidNamespacePath(), ".", "/"), "/OneginiConfigModel.java")
+	}
+	return modelPath
 }
 
 func (config *Config) getAndroidSecurityControllerPath() string {
-	return path.Join(getPlatformSpecificAndroidClasspathPath(config), "SecurityController.java")
+	modelPath := path.Join(getPlatformSpecificAndroidClasspathPath(config), "SecurityController.java")
+	// if modelPath has no package name, check namespace property in build.gradle
+	if strings.HasSuffix(modelPath, "java/SecurityController.java") {
+		modelPath = strings.TrimSuffix(modelPath, "SecurityController.java")
+		modelPath = path.Join(modelPath, strings.ReplaceAll(config.getAndroidNamespacePath(), ".", "/"), "/SecurityController.java")
+	}
+	return modelPath
 }
 
 func (config *Config) getAndroidClasspathPath() string {
 	return path.Join(getPlatformSpecificAndroidClasspathPath(config))
 }
 
+func (config *Config) getAndroidNamespacePath() string {
+	gradleFilePath := path.Join(config.AppDir, config.AppTarget, "build.gradle")
+	gradleContent, err := ioutil.ReadFile(gradleFilePath)
+	if err != nil {
+		fmt.Println("Error during reading gradle file", err)
+	}
+	namespaceRegexMatches := regexp.MustCompile(`(?m)^\s*namespace\s+'([^']+)'\s*$`).FindStringSubmatch(string(gradleContent))
+	if len(namespaceRegexMatches) == 2 {
+		return namespaceRegexMatches[1]
+	} else {
```

Comment:
why we need to check for len == 2 ? Is it becuase of the groups matching of te regex?
Maybe this regex would work: `(?m)^\s*namespace\s+['"][^']+['"]\s*$` ? Did some testing of it on regex101.com, but I wasn't able to test it in the configurator because of building issues :(

Reply:
I have adjusted regex to cover both `"` and `''`, I've tried to also just return the second match, which is the proper one, but to no avail :/